bin/kano-os-recovery

```diff
@@ -40,19 +40,18 @@ if [ $system_stable != 1 ] && [ "$mode" == "--recovery" ]; then
 
     echo "Recovery is starting now..."
 
-    # mount file systems read-write (rootfs and boot are needed by dpkg)
-    /bin/mount /dev/mmcblk0p1 /boot
-    /bin/mount -o remount,rw /dev/mmcblk0p2 /
+    # Prevent apt recovery tasks from starting daemons from package postinst scripts
+    # FIXME: Can we tell systemd to do this for us?
+    /bin/mount --bind /bin/true /usr/sbin/invoke-rc.d
```

Comment:
I think this would still require `MountAPIVFS=true` in the .service file, as discussed in the previous PR.

However, from the `invoke-rc.d` man page: 
"invoke-rc.d introduces the concept of a policy layer which is used to verify if an init script should be run or not"
This suggests there might be a cleaner way of doing this: write an `/usr/sbin/policy-rc.d` script https://people.debian.org/~hmh/invokerc.d-policyrc.d-specification.txt

Reply:
So we can either write a `policy-rc.d` which can signal in some way whether the recovery is running, maybe with an enviroment variable, or use the mount bind as above but on `policy-rc.d` instead of `invoke-rc.d`

Reply:
for the records - I tried the policy override system and it works well for dnsmasq, but it stucks the system again when setting up dbus.